Kitodo/src/main/java/org/kitodo/services/data/ProcessService.java

```diff
@@ -2156,17 +2156,20 @@ public void imageDownload(Process process, URI userHome, String atsPpnBand, fina
                 /*
                  * wenn kein Agora-Import, dann den Ordner mit Benutzerberechtigung neu anlegen
                  */
-                User myUser = (User) Helper.getManagedBeanValue("#{LoginForm.myBenutzer}");
+                User user = Helper.getCurrentUser();
                 try {
-                    fileService.createDirectoryForUser(zielTif, myUser.getLogin());
+                    if (user != null) {
+                        fileService.createDirectoryForUser(zielTif, user.getLogin());
+                    } else {
+                        throw new IOException("noLoggedUser");
```

Comment:
An exception messages must not be translatable and they are not translated until now.